tgui/packages/tgui/debug/KitchenSink.jsx

```diff
@@ -22,8 +22,8 @@ const getStories = () => r.keys().map((path) => r(path));
 
 export const KitchenSink = (props) => {
   const { panel } = props;
-  const [theme] = useLocalState('kitchenSinkTheme');
```

Comment:
Need a 2nd opinion on this

Reply:
interesting. this line and the one below are being used as globals. i'd leave them for now

Reply:
Applied with https://github.com/tgstation/tgstation/pull/80459/commits/6ec5fc718fe81976fc95fc9468452aa249172ffe